component/image/image.html

```diff
@@ -1,58 +1,70 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Image-component</title>
-    <link rel="stylesheet" href="image.css">
+    <title>BOOSTWEB-UI | Image</title>
```

Comment:
Better to use meta tag with your name in the content

Reply:
Better to use for SEO. I have seen the MAs commenting about it in their Reviews.

Reply:
Not compulsory I guess. But, better to have